<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartReceiverView</class>
 <widget class="QWidget" name="StartReceiverView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="receiverLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>U-Blox Receiver Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="receiverStatusLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>[Not Detected]</string>
   </property>
  </widget>
  <widget class="QComboBox" name="startTypeSelect">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Select Start Type</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cold</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Warm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hot</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
